istioctl/cmd/tag.go

```diff
@@ -321,6 +330,14 @@ func setTag(ctx context.Context, kubeClient kube.ExtendedClient, tag, revision s
 	if webhookName != "" {
 		tagWhYAML = renameTagWebhookConfiguration(tagWhYAML, tag, webhookName)
 	}
+
+	resName := webhookName
+	if resName == "" {
+		resName = fmt.Sprintf("%s-%s", "istio-revision-tag", tag)
+	}
+	if err := analyzeWebhook(resName, tagWhYAML, kubeClient.RESTConfig()); err != nil {
```

Comment:
nit: if user sets `skipConfirmation` and they're generating the webhook we probably want to warn but go ahead and generate (they might be generating for later application)